### PR TITLE
[INS-3511] [Products] Import does not work

### PR DESCRIPTION
**Pull request by RockMother**
Changed parameter for the import. 
Parameters moved from URL to the request body.

### CODE REVIEW

**Review by aehrenthal — APPROVED** [review]

File: sell/sell-api.yaml [hunk 1]
```diff
@@ -1768,17 +1768,13 @@ paths:
       summary: Import a file
       description: Use this endpoint to upload a file containing entities to Inperium. For example, you can upload your client data exported from other CRMs and that was previously stored in spreadsheets.
```
Comment (olyavertwist) [inline sell/sell-api.yaml:1769]: description: Use this endpoint to upload a file containing entities to Inperium. For example, you can upload your client data exported from other CRMs or data that was previously stored in spreadsheets.

File: sell/sell-api.yaml [hunk 2]
```diff
@@ -5092,6 +5088,25 @@ components:
         paging:
           $ref: "#/components/schemas/PageAndSort"
       description: The list of import events.
+    ImportRequest:
+      type: object
```
Comment (olyavertwist) [inline sell/sell-api.yaml:5092]: description: The request object containing import data.

File: sell/sell-api.yaml [hunk 3]
```diff
@@ -5092,6 +5088,25 @@ components:
...
+          type: array
+          description: This parameter defines mapping between incoming data and Inperium Sell properties.
+          items:
+            type: object
+            properties:
+              column:
+                description: The caption of the column from the file imported.
+                type: string
+              propertyId:
+                description: The id of the property.
```
Comment (olyavertwist) [inline sell/sell-api.yaml:5104]: description: The ID of the property.

**Review by olyavertwist — CHANGES_REQUESTED** [review]
Really minor changes.

**Review by olyavertwist — APPROVED** [review]

**Pull request merged**